src/Filter.BBox.js

```diff
@@ -0,0 +1,15 @@
+L.Filter.BBox = L.Filter.extend({
+  append: function(bbox, geometryField, crs) {
+    var filterBBox = L.XmlUtil.createElementNS('ogc:BBOX');
+    filterBBox.appendChild(L.XmlUtil.createElementNS('ogc:PropertyName', {}, {value: geometryField}));
+
+    var envelope = L.XmlUtil.createElementNS('gml:Envelope', {srsName: crs.code});
+    envelope.appendChild(L.XmlUtil.createElementNS('gml:lowerCorner', {}, {value: bbox.getSouthWest().lng + ' ' + bbox.getSouthWest().lat}));
```

Comment:
And again, this will not work. Such as you use [latlngbounds](http://leafletjs.com/reference.html#latlngbounds), you must do [project](http://leafletjs.com/reference.html#icrs-project) for bbox corners:
```javascript
var projectedSW = crs.project(bbox.getSouthWest());
var projectedNE = crs.project(bbox.getNorthEast());

var envelope = L.XmlUtil.createElementNS('gml:Envelope', {srsName: crs.code});
envelope.appendChild(L.XmlUtil.createElementNS('gml:lowerCorner', {}, {value: projectedSW.x + ' ' + projectedSW.y}));
...
```